backends/magma/ceed-magma.h

```diff
@@ -22,13 +22,51 @@
 #define USE_MAGMA_BATCH
 #define USE_MAGMA_BATCH2
 #define USE_MAGMA_BATCH3
+#define USE_MAGMA_BATCH4
 
 void magma_dtensor_contract(Ceed ceed,
                             CeedInt A, CeedInt B, CeedInt C, CeedInt J,
                             const CeedScalar *t, CeedTransposeMode tmode,
                             const CeedInt Add,
                             const CeedScalar *u, CeedScalar *v);
 
+static int magma_dbasisApply_batched(
```

Comment:
Style question: why a static function in the header?

Reply:
This declaration is actually unnecessary. It has been removed